AdobeStockImageAdminUi/view/adminhtml/web/js/components/grid/column/overlay.js

```diff
@@ -79,14 +103,24 @@ define([
         },
 
         /**
-         * Get all ids from data provider
+         * Checks if user is logged in and authorized
+         *
+         * @returns {Boolean}
+         */
+        isUserNotAuthorized: function() {
```

Comment:
I'd prefer to revert the function to have a better naming
```suggestion
        isUserAuthorized: function() {
```